build/report.py

```diff
@@ -150,6 +130,7 @@ def writeReport(filename, results, serverName, namespace, checksInfo):
 parser.add_argument("-l", help="log file name", type=str, default="-")
 parser.add_argument("-o", help="output file name", type=str, default="report.html")
 parser.add_argument('-n', help="namespace", type=str, default="default")
+parser.add_argument('-c', help="cluster name", type=str, default="default cluser name")
```

Comment:
i think that default is incorrect. perhaps default to `silver`?

Reply:
sounds good. I shall make it so.